docs/components/ListItems.md

```diff
@@ -49,38 +50,36 @@ Please note that there is no start padding with wide images.
 
 The library offers the `OdsListItem` composable to display lists items.
 
-**Leading icon:**
-
-To specify an icon type, use the `Modifier.iconType` method on the `OdsListItem` modifier and call `OdsListItemScope.OdsListItemIcon` in the `icon` lambda.
-
-**Trailing element:**
-
-The `OdsListItem` composable allows you to display an `OdsListItemTrailing` (Checkbox, Switch, RadioButton, Icon or a Caption text) as trailing element. If this does not meet your
-needs and only in this case, you can use the `OdsListItem` method signature which accept any Composable as trailing.
-
-Note: The first signature (with `OdsListItemTrailing`) automatically manages accessibility, if you use the second one, don't forget to manage it on your side.
-
-**Dividers:**
-
-A divider can also be displayed at the bottom of the list item using the `Modifier.divider` method on the `OdsListItem` modifier.
+The `OdsListItem` composable allows you to display a leading icon using the `icon` parameter of the `OdsListItem` method, as well as a trailing element (either a checkbox, a switch, a radio button, an icon or a caption text) using the `trailing`parameter.
 
 ```kotlin
 OdsListItem(
-    modifier = Modifier
-        .clickable { doSomething() }
-        .iconType(OdsListItemIconType.Icon)
-        .divider(),
+    modifier = Modifier.clickable { doSomething() },
     text = "Primary text",
-    icon = {
-        OdsListItemIcon(
-            painter = painterResource(id = R.drawable.ic_heart),
-            contentDescription = "Heart"
-        )
-    },
-    trailing = OdsCheckboxTrailing(checked = itemChecked)
+    icon = OdsListItemIcon(
+        OdsListItemIconType.Icon,
+        painterResource(id = R.drawable.ic_heart),
+        "Heart"
+    ),
+    trailing = OdsListItemTrailingCheckbox(checked, { checked != checked }),
+    divider = true
 )
 ```
 
+##### OdsListItem API
+
+Parameter | Default&nbsp;value | Description
+-- | -- | --
+`text: String` | | The primary text of the list item.
```

Comment:
I will remove them for now for consistency. But I hope we will manage to automatically update the markdown files with the KDoc, this will fix these kind of issues.